examples/src/test/kotlin/ChangeStreamsTest.kt

```diff
@@ -136,6 +137,37 @@ internal class ChangeStreamsTest {
 
     }
 
+    // This test is getting ignored because the feature is only enabled for deployments on MongoDB 7.0
```

Comment:
Suggestion:

No change necessary, but I think this could be stated more concisely:

```suggestion
    // Ignore annotation added because this function requires a MongoDB 7.0 deployment
```